app/models/User.coffee

```diff
@@ -522,6 +522,7 @@ module.exports = class User extends CocoModel
   hideFooter: -> @isTarena()
   useGoogleClassroom: -> not (features?.chinaUx ? false) and me.get('gplusID')?   # if signed in using google SSO
   useGoogleAnalytics: -> not (features?.chinaInfra ? false)
+  isCocoChinaInfra: -> features?.chinaInfra ? false
```

Comment:
Should we rename this flag to be specific to video hosting?
`isCocoChinaInfra` seems a duplicate of `features.chinaInfra`

Reply:
Great point. Changing both side to reflect intention better.